libs/sdk-ui-dashboard/src/presentation/scheduledEmail/DefaultScheduledEmailDialog/ScheduledMailDialogRenderer/RepeatSelect/RepeatFrequencySelect.tsx

```diff
@@ -5,11 +5,27 @@ import { Dropdown, DropdownList, DropdownButton, SingleSelectListItem } from "@g
 import invariant from "ts-invariant";
 
 import { IDropdownItem } from "../../interfaces";
-import { DEFAULT_DROPDOWN_ALIGN_POINTS, DEFAULT_DROPDOWN_ZINDEX, FREQUENCY_TYPE } from "../../constants";
+import {
+    DEFAULT_DROPDOWN_ALIGN_POINTS,
+    DEFAULT_DROPDOWN_ZINDEX,
+    FREQUENCY_TYPE,
+    REPEAT_FREQUENCIES,
+} from "../../constants";
 import { messages } from "../../../../../locales";
 
 const DROPDOWN_WIDTH = 100;
 
+const getLocalizationKey = (id: string): { id: string } => {
+    switch (id) {
+        case REPEAT_FREQUENCIES.DAY:
+            return messages.scheduleDialogEmailRepeatsFrequencies_day;
+        case REPEAT_FREQUENCIES.MONTH:
+            return messages.scheduleDialogEmailRepeatsFrequencies_month;
+        default:
+            return messages.scheduleDialogEmailRepeatsFrequencies_week;
```

Comment:
Here as well, please: add it to its own `case` branch and throw exception in `default` branch instead.

Reply:
Added.